cache_ui/cinfo.go

```diff
@@ -0,0 +1,122 @@
+package cache_ui
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"hash/crc32"
+
+	"github.com/pkg/errors"
+)
+
+type (
+	CInfo struct {
```

Comment:
Do not export the structs
```suggestion
	cInfo struct {
```